scripts/ipkg-build

```diff
@@ -68,6 +68,40 @@ pkg_appears_sane() {
 	return $PKG_ERROR
 }
 
+resolve_file_mode_id() {
+	type="$1"
+	name="$2"
+	position=1
+	if [ "$type" = "group" ]; then
+		position=2
+	fi
+
+	# root is always 0
+	if [ "$name" = "root" ]; then
+		echo 0
+		exit 0
+	fi
+
+	# return numeric names
+	if [ "$name" -eq "$name" ]; then
```

Comment:
This is not a nice way to test for numeric content of a variable, it will result in many errors like
```bash: [: foo: integer expression expected```
Either use `grep` or 
```suggestion
	if [ "$name" -eq "$name" 2>/dev/null ]; then
```